### ISSUE TITLE
CI failing due to puppeteer@3.0.0

### ISSUE THREAD

**Issue by fcollonval** [jupyterlab/extension-cookiecutter-ts#60]
Since 2 days, the check browser test is failing due to the usage of puppeteer@3.0.0.

An issue is opened there: https://github.com/puppeteer/puppeteer/issues/5674

**Comment by jtpio** [comment]
Thanks @fcollonval for opening the issue.

Do you think it would be worth applying a fix by pinning `puppeteer` to `^2` in the [browser check](https://github.com/jupyterlab/jupyterlab/blob/f0153e0258b32674c9aec106383ddf7b618cebab/jupyterlab/browser_check.py#L147)?

**Comment by blink1073** [comment]
I'm hesitant to make a new set of releases for this, at the very least I don't have the bandwidth until Wednesday.

**Comment by jtpio** [comment]
As mentioned in https://github.com/puppeteer/puppeteer/issues/5674, on possible workaround for now is to add the following step to the GitHub Actions workflow:

```yaml
    - name: TMP fix - install libgbm1
      run: sudo apt install -y libgbm1
```

**Comment by fcollonval** [comment]
As per https://github.com/actions/virtual-environments/issues/732, the upstream Ubuntu image will be update to add that new dependency. Should we update the repo with the temporary fix or just wait for an update of image?

**Comment by blink1073** [comment]
I released JupyterLab 2.1.1 with a pinned version of puppeteer.  I tried to run our failing CI build, but it is currently still picking up 2.1.0 in the install step.

**Comment by fcollonval** [comment]
Thanks @blink1073 !!
The CI is now passing.
